test/test_errno.rs

```diff
@@ -0,0 +1,8 @@
+use nix::errno::Errno;
+
+#[test]
+fn errno_set_and_read() {
+    Errno::clear();
```

Comment:
This test doesn't actually verify that clear does what it's supposed to.  You should probably have one test for set and another for clear.

Reply:
Done

Reply:
But why leave the Errno::clear there on line 5?  You should just delete it.

Reply:
Oh, yeah, set will override it anyway, not sure why I even added it in the first place :facepalm: 
Done